sympy/interactive/printing.py

```diff
@@ -389,14 +389,15 @@ def init_printing(pretty_print=True, order=None, use_unicode=None,
         else:
             # This will be True if we are in the qtconsole or notebook
             if not isinstance(ip, (InteractiveConsole, TerminalInteractiveShell)) \
-                    and 'ipython-console' not in ''.join(sys.argv):
+                    and 'ipython-console' not in ''.join(sys.argv)
```

Comment:
You missed a colon at the end. So, the build is failing because of `SyntaxError`